presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -242,10 +241,10 @@ protected Node visitNode(Node node, Void context)
                     .findSingle();
 
             if (!scanNode.isPresent()) {
-                return new Constraint<>(TupleDomain.none(), bindings -> true);
```

Comment:
It should be `bindings -> false`. Please change in a separate (preparatory) commit